### ISSUE TITLE
possible to change the default log level?

### ISSUE THREAD

**Issue by ondras** [denoland/deno#3866]
My component is happy to use the built-in logger:

```js
// component.ts
import * as log from "std/log/mod.ts";
log.debug("debug stuff");
```

...and I would like to be able to centrally configure the log level for all components (preferrably by parsing command line arguments or so). I can asynchronously reconfigure the default logger using `log.setup()`, but this requires me to completely define a new set of loggers/handlers. My only need is to change the default loglevel used by `log.*`. How can I do that?

**Comment by bartlomieju** [comment]
@ondras `log` will keep old loggers and handlers when you configure it:
https://github.com/denoland/deno/blob/f168597b7ab81afda3bf4749a81c360d364e7cf1/std/log/mod.ts#L76-L119

You should be able to do:
```ts
await log.setup({
  loggers: {
    level: 'ERROR',
    handlers: ["default"]
  },
})
```

**Comment by ondras** [comment]
> You should be able to do:

This does not seem to work for me. First, `loggers` and `handlers` shall be two distinct top-level properties. Second, the `setup()` call will re-create all loggers (https://github.com/denoland/deno/blob/master/std/log/mod.ts#L114), populating them with supplied `handlers`.

So in the end, I am unable to reconfigure the default logger without repopulating the whole loggers/handlers namespace.

For reference, this is what I am trying (and failing):

```js
await log.setup({
  loggers: {
    default: {
      level: "DEBUG"
    }
  }
})
```

**Comment by bartlomieju** [comment]
This issue should have been fixed in #5341
